docs/reference/user/provisioning/scim-via-curl.md

```diff
@@ -0,0 +1,172 @@
+# Using the SCIM API with curl {#RefScimViaCurl}
+
+_Author: Matthias Fischmann_
+
+---
+
+This page shows you how to communicate with the wire backend through
+the [SCIM API](http://www.simplecloud.info/) by example.  All examples
+are [curl](https://curl.haxx.se/) (in bash syntax).
+
+If you want to dive into the backend code, start [reading here in our
+backend](https://github.com/wireapp/wire-server/blob/develop/services/spar/src/Spar/Scim.hs)
+and [our hsim library](https://github.com/wireapp/hscim).
+
+
+## Creating an SCIM token
+
+First, we need a little shell environment:
+
+```
+export WIRE_BACKEND=prod-nginz-https.wire.com
```

Comment:
Perhaps change it to `https://prod-nginz-https.wire.com` so that "https" would not be hardcoded into bash snippets. (Somebody might want to try all that locally and then it'd be "http".)